notebooks/start-a-business-checker/users_visiting_two_start_a_business_content_items.sql

```diff
@@ -0,0 +1,13 @@
+
+select count(distinct Sessionid) as Sessions, CountofPages
```

Comment:
```suggestion
select count(Sessionid) as Sessions, CountofPages
```

`DISTINCT` is not needed as your previous step produces one single row per sessionId